test/twine_test.coffee

```diff
@@ -277,10 +277,10 @@ suite "Twine", ->
       node = setupView(testView, context = fn: @spy())
       data = {test: 'bla123'}
 
-      $(node).trigger 'submit', data
```

Comment:
The test fails using jQ trigger too. I rewrote `triggerEvent` to try and get the test to pass